celesta-sql/src/main/java/ru/curs/celesta/score/CelestaSerializer.java

```diff
@@ -0,0 +1,563 @@
+package ru.curs.celesta.score;
```

Comment:
Nice, after all the `save` methods moved to this class, this started to resemble "Visitor" pattern. This can be further refactored, but I like it already!